options.go

```diff
@@ -310,6 +311,14 @@ func WithGCHeapDump(min int, diff int, abs int) Option {
 	})
 }
 
+// WithCPUCore overwrite the system level CPU core number when it > 0.
+func WithCPUCore(cpuCore float64) Option {
```

Comment:
I thought this function should not be called through holmes.Set() when the application is running.
because it will break the continuous of metrics.
Is there any way to forbid it?

Reply:
yeah, it's not a good idea to call it on fly in normal cases.
Just added a comment for it, I think it's good enough now.